### DISCUSSION TITLE
VirtualDisplay Driver should be loaded by default when there is no video output detected.

### DISCUSSION THREAD

**Discussion by Adephx** [LizardByte/community#584] [Sunshine]
### Select Topic Area

Feature Request

### Body

This should be included by default, especially for laptop users, so you don't have to default to use a HDMI dummy plug. The Beta version works for my dual-gpu laptop:

https://github.com/VirtualDisplay/Virtual-Display-Driver/releases/tag/24.10.27

**Comment by DULLONEX** [comment]
me too.
It bothers me
